pypsa/networkclustering.py

```diff
@@ -207,9 +207,24 @@ def aggregatelinegroup(l):
     linemap_n = interlines_n.join(lines['name'], on=['bus0_s', 'bus1_s'])['name']
     linemap = pd.concat((linemap_p,linemap_n), sort=False)
 
-    return lines, linemap_p, linemap_n, linemap
+    lines_t = dict()
 
-def get_buses_linemap_and_lines(network, busmap, line_length_factor=1.0, bus_strategies=dict()):
+    if with_time:
+        for attr, df in network.lines_t.items():
+            df.columns = df.columns.astype(str)
+            lines_agg_b = df.columns.to_series().map(linemap).dropna()
+            df_agg = df.loc[:, lines_agg_b.index]
+            if not df_agg.empty:
+                pnl_df = df_agg.mul(
+                    network.lines.groupby(linemap).s_nom.apply(
+                        lambda grp: grp/grp.sum()
+                        )).groupby(linemap, axis=1).sum()
+                pnl_df.columns = _flatten_multiindex(pnl_df.columns).rename("name")
+                lines_t[attr] = pnl_df
```

Comment:
```suggestion
    if with_time:
        for attr, df in network.lines_t.items():
            lines_agg_b = df.columns.to_series().map(linemap).dropna()
            df_agg = df.loc[:, lines_agg_b.index]
            if not df_agg.empty:
                if (attr == 's_max_pu') or (attr == "s_min_pu"):
                    weighting = network.lines.groupby(linemap).s_nom.transform(_normed)
                    df_agg = df_agg.multiply(weighting.loc[df_agg.columns], axis=1)
                pnl_df = df_agg.groupby(linemap, axis=1).sum()
                pnl_df.columns = _flatten_multiindex(pnl_df.columns).rename("name")
                lines_t[attr] = pnl_df

```

Reply:
@ClaraBuettner makes sense. indeed, I merged your contribution via a further PR. Thanks for tackling this.